package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sphere-node-product-csv-sync",
   "description": "Import and export your products from/to your SPHERE.IO project.",
-  "version": "0.6.8",
+  "version": "0.6.11",
```

Comment:
We use `grunt release` to bump the version - this will also publish the module on npmjs.org